src/NuGet.Server/Web.config

```diff
@@ -43,7 +43,13 @@
     <!--
     Set enableFrameworkFiltering to true to enable filtering packages by their supported frameworks during search.
     -->
-    <add key="enableFrameworkFiltering" value="false"/>
+    <add key="enableFrameworkFiltering" value="false" />
+    
+	<!--
+	If value is setted to 0, there are no retention otherwise the retention will be setted to the defined value
+	-->
+	<add key="retentionRule" value="0" />
```

Comment:
I'd rename the setting to `PackageVersionRetention`, and provide instructions in comments:
```
<!--
A numeric value defining the number of package versions to retain on the server.
When the defined number of versions of a package exceeds this value, the oldest versions will be removed until the defined maximum number of package versions is reached.
To disable this feature and retain all versions, set the value to 0.
-->
```